lib/twine-ui/src/styles/spacing.ts

```diff
@@ -0,0 +1,3 @@
+export enum PaddingEnum {
```

Comment:
This was added a while ago, is this still consistent with (1) any updates to the design system and (2) the equivalent components in the dashboard?